ngraph/core/include/openvino/runtime/tensor.hpp

```diff
@@ -148,6 +148,15 @@ class OPENVINO_API Tensor {
      * @return `true` if current Tensor object is initialized, `false` - otherwise
      */
     explicit operator bool() const noexcept;
+
```

Comment:
let's also add a check for `data()` method that it throws exception for case when impl is remote blob